docs/_docs/memory-configuration/data-regions.adoc

```diff
@@ -140,7 +192,9 @@ control.bat --warm-up --stop --yes
 ----
 --
 
-To stop the warming up using JMX, use the method below:
+
+To disable the warm-up using JMX, use the method:
```

Comment:
We can stop it if the user needs it, for example it takes too long.

Reply:
Fixed.